master/getting-started/kubernetes/upgrade/upgrade.md

```diff
@@ -42,30 +58,68 @@ Once you have upgraded {{site.prodname}}, you can [complete the upgrade](#comple
    calico-node-w92wk                          1/2       Running   0          3m
    ```
 
-   > **Note**: The {{site.noderunning}} pods will report `1/2` in the `READY` column, as shown.
+   > **Tip**: The {{site.noderunning}} pods will report `1/2` in the `READY` column, as shown.
+   {: .alert .alert-success}
+
+   > **Note**: If an error occurs during the upgrade, refer to 
+   > [Downgrading Calico](/{{page.version}}/getting-started/kubernetes/upgrade/downgrade).
    {: .alert .alert-info}
    
-1. After waiting some time and ensuring that the upgrade succeeded and no problems ensued,
-   continue to [Completing the upgrade](#completing-the-upgrade).
+1. Remove any existing `calicoctl` instances and install the new `calicoctl`.
 
-   If an error occurs during the upgrade, refer to [Downgrading Calico](/{{page.version}}/getting-started/kubernetes/upgrade/downgrade).
+1. Congratulations! You have upgraded to {{site.prodname}} {{page.version}}.
 
 
-## Upgrading a custom installation
+## Upgrading a self-hosted installation that uses the etcd datastore
 
-_Docs for this coming soon!_
+1. [Refer to the v3.0 documentation and obtain the manifest that matches your installation
+   method.](https://docs.projectcalico.org/v3.0/getting-started/kubernetes/installation/hosted/)
 
-## Completing the upgrade
+1. Use the following command to initiate a rolling update, after replacing 
+   `<v3-manifest>` with the name of the manifest file obtained in the previous step.
 
-1. When all of your calico/node instances and orchestrator plugins are running v3.0.x 
-   then complete the upgrade by running `calico-upgrade complete`. After this, you can
-   once again schedule pods and make changes to configuration and policy. 
+   ```
+   kubectl apply -f <v3-manifest>
+   ```
+   
+1. Watch the status of the upgrade as follows.
 
-   If you experience errors after running `calico-upgrade complete`, such as an inability
-   to schedule pods, [downgrade Calico as soon as possible](/{{page.version}}/getting-started/kubernetes/upgrade/downgrade).
+   ```
+   watch kubectl get pods -n kube-system
+   ```
+   
+   Verify that the status of all {{site.prodname}} pods indicate `Running`.
+
+   ```
+   calico-kube-controllers-6d4b9d6b5b-wlkfj   1/1       Running   0          3m
+   calico-node-hvvg8                          1/2       Running   0          3m
+   calico-node-vm8kh                          1/2       Running   0          3m
+   calico-node-w92wk                          1/2       Running   0          3m
+   ```
+
+   > **Tip**: The {{site.noderunning}} pods will report `1/2` in the `READY` column, as shown.
+   {: .alert .alert-success}
+
+   > **Note**: If an error occurs during the upgrade, refer to 
+   > [Downgrading Calico](/{{page.version}}/getting-started/kubernetes/upgrade/downgrade).
+   {: .alert .alert-info}
+   
+1. We recommend waiting for some time and really ensuring that the upgrade succeeded 
```

Comment:
might make sense to give instructions for verifying when the node has been upgraded ...
`kubectl exec -n kube-system calico-node-hvvg8 versions` should report v3.0.0.